{
  "uid" : "388dd3762bb81811",
  "name" : "attributes",
  "fullName" : "Products API tests: List All Products List all products.attributes",
  "historyId" : "f1190e9e821d63747d1c351b6fff157c",
  "time" : {
    "start" : 1655092558063,
    "stop" : 1655092558064,
    "duration" : 1
  },
  "description" : "",
  "descriptionHtml" : "",
  "status" : "failed",
  "statusMessage" : "Error: connect ECONNREFUSED 162.241.16.11:443",
  "statusTrace" : "Error: connect ECONNREFUSED 162.241.16.11:443\n                    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1187:16)",
  "flaky" : false,
  "newFailed" : false,
  "newBroken" : false,
  "newPassed" : false,
  "retriesCount" : 13,
  "retriesStatusChange" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Products API tests: List All Products List all products"
  }, {
    "name" : "testClass",
    "value" : "Products API tests: List All Products List all products"
  }, {
    "name" : "testMethod",
    "value" : "attributes"
  }, {
    "name" : "package",
    "value" : "Products API tests: List All Products List all products"
  }, {
    "name" : "resultFormat",
    "value" : "allure1"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "6dd8540a84f6e79a",
      "status" : "failed",
      "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
      "time" : {
        "start" : 1655009760664,
        "stop" : 1655009760665,
        "duration" : 1
      }
    }, {
      "uid" : "5fd2fd28406e24e0",
      "status" : "failed",
      "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
      "time" : {
        "start" : 1654923812191,
        "stop" : 1654923812193,
        "duration" : 2
      }
    }, {
      "uid" : "2af9dc039a0ee139",
      "status" : "failed",
      "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
      "time" : {
        "start" : 1654837529063,
        "stop" : 1654837529064,
        "duration" : 1
      }
    }, {
      "uid" : "125dbd9ee271a262",
      "status" : "failed",
      "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
      "time" : {
        "start" : 1654751024579,
        "stop" : 1654751024580,
        "duration" : 1
      }
    }, {
      "uid" : "a4d2b3f44f7bc213",
      "status" : "failed",
      "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
      "time" : {
        "start" : 1654660393195,
        "stop" : 1654660393195,
        "duration" : 0
      }
    }, {
      "uid" : "844ea98f7b02732e",
      "status" : "failed",
      "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
      "time" : {
        "start" : 1654578808566,
        "stop" : 1654578808566,
        "duration" : 0
      }
    }, {
      "uid" : "d74f8cc21f44b6c2",
      "status" : "failed",
      "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
      "time" : {
        "start" : 1654491950570,
        "stop" : 1654491950570,
        "duration" : 0
      }
    }, {
      "uid" : "ee36585ee7993b53",
      "status" : "failed",
      "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
      "time" : {
        "start" : 1654405016201,
        "stop" : 1654405016201,
        "duration" : 0
      }
    }, {
      "uid" : "49e75e6f378c65fb",
      "status" : "failed",
      "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
      "time" : {
        "start" : 1654318655497,
        "stop" : 1654318655497,
        "duration" : 0
      }
    }, {
      "uid" : "887350cd79174bfa",
      "status" : "failed",
      "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
      "time" : {
        "start" : 1654232271345,
        "stop" : 1654232271346,
        "duration" : 1
      }
    }, {
      "uid" : "13278ce79c5fffe1",
      "status" : "failed",
      "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
      "time" : {
        "start" : 1654146606606,
        "stop" : 1654146606606,
        "duration" : 0
      }
    }, {
      "uid" : "445310d8de9482ee",
      "status" : "failed",
      "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
      "time" : {
        "start" : 1654060820232,
        "stop" : 1654060820233,
        "duration" : 1
      }
    }, {
      "uid" : "62b8175c8388128f",
      "status" : "failed",
      "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
      "time" : {
        "start" : 1653893610419,
        "stop" : 1653893610420,
        "duration" : 1
      }
    } ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 14,
        "broken" : 0,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 14
      },
      "items" : [ {
        "uid" : "2799077e98ded442",
        "status" : "failed",
        "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
        "time" : {
          "start" : 1655009760664,
          "stop" : 1655009760665,
          "duration" : 1
        }
      }, {
        "uid" : "275f35dc158c9a50",
        "status" : "failed",
        "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
        "time" : {
          "start" : 1654923812191,
          "stop" : 1654923812193,
          "duration" : 2
        }
      }, {
        "uid" : "de9fda2aaa55a4fd",
        "status" : "failed",
        "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
        "time" : {
          "start" : 1654837529063,
          "stop" : 1654837529064,
          "duration" : 1
        }
      }, {
        "uid" : "404ca70b78521fa4",
        "status" : "failed",
        "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
        "time" : {
          "start" : 1654751024579,
          "stop" : 1654751024580,
          "duration" : 1
        }
      }, {
        "uid" : "85ad43094879d8d5",
        "status" : "failed",
        "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
        "time" : {
          "start" : 1654660393195,
          "stop" : 1654660393195,
          "duration" : 0
        }
      }, {
        "uid" : "71fb09af9d6e3236",
        "status" : "failed",
        "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
        "time" : {
          "start" : 1654578808566,
          "stop" : 1654578808566,
          "duration" : 0
        }
      }, {
        "uid" : "697e7750ae4e6e97",
        "status" : "failed",
        "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
        "time" : {
          "start" : 1654491950570,
          "stop" : 1654491950570,
          "duration" : 0
        }
      }, {
        "uid" : "8778a4580b981b2b",
        "status" : "failed",
        "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
        "time" : {
          "start" : 1654405016201,
          "stop" : 1654405016201,
          "duration" : 0
        }
      }, {
        "uid" : "e9ad3e26b53772dd",
        "status" : "failed",
        "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
        "time" : {
          "start" : 1654318655497,
          "stop" : 1654318655497,
          "duration" : 0
        }
      }, {
        "uid" : "6339a4a2b4c764af",
        "status" : "failed",
        "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
        "time" : {
          "start" : 1654232271345,
          "stop" : 1654232271346,
          "duration" : 1
        }
      }, {
        "uid" : "210d929693541460",
        "status" : "failed",
        "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
        "time" : {
          "start" : 1654146606606,
          "stop" : 1654146606606,
          "duration" : 0
        }
      }, {
        "uid" : "9d0cf0980132fe1e",
        "status" : "failed",
        "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
        "time" : {
          "start" : 1654060820232,
          "stop" : 1654060820233,
          "duration" : 1
        }
      }, {
        "uid" : "b273180212d9d8f6",
        "status" : "failed",
        "statusDetails" : "Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.",
        "time" : {
          "start" : 1653893610419,
          "stop" : 1653893610420,
          "duration" : 1
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "388dd3762bb81811.json",
  "parameterValues" : [ ]
}